src/olympia/migrations/1100-add-disposable-emails-domain-restriction.sql

```diff
@@ -0,0 +1,4544 @@
+CREATE TABLE `users_disposable_email_domain_restriction` (
+    `id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `domain` varchar(255) NOT NULL UNIQUE
+);
+
+INSERT INTO `users_disposable_email_domain_restriction` (`domain`, `created`, `modified`) VALUES
```

Comment:
Would it be a good idea to just create the table during the migration, and load the data via an after-push script? From a push perspective, the less to do during the migration the better.

Reply:
Locally inserting those rows was almost instantaneous so I figured it wasn't much of a problem. Was it long on stage?

Reply:
Nah, it's not a big deal and it's below a second in this case. I was more wondering from a best/standard practice perspective.

Reply:
It's quite a rare occurrence so we don't have established best practices for it. I'd have probably done this via a script if it was addons/versions/files/users table, like we did when we mass-disabled legacy add-ons for instance.